lib/model/rwfolder.go

```diff
@@ -780,13 +778,9 @@ func (f *rwFolder) renameFile(source, target protocol.FileInfo) {
 	from := filepath.Join(f.dir, source.Name)
 	to := filepath.Join(f.dir, target.Name)
 
-	if f.versioner != nil {
```

Comment:
This change is a no-go. It transforms a rename into a copy + delete when we have the default versioner (no versioning).